src/resources/elements/linkfield.js

```diff
@@ -45,6 +45,21 @@ export class Linkfield extends Field {
     return field;
   }
 
+  resolvePath(path) {
```

Comment:
Is/was the problem on line 48 or somewhere else? I don't see anything wrong with line 48.